install-openjdk-11.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e 
+
+export DEBIAN_FRONTEND=noninteractive
+
+echo "deb http://deb.debian.org/debian bullseye main" > /etc/apt/sources.list.d/openjdk-11.list
```

Comment:
What is this script doing? Are we just installing the debian 11 (bullseye) packing on debian 12 (bookworm)? This is probably not safe.

Reply:
```diff
diff --git a/Dockerfile b/Dockerfile
index d0600dd..793c60f 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -12,10 +12,14 @@ RUN set -x \
   ;

 # Install JDK11
-COPY install-openjdk-11.sh /
 RUN set -x \
   && mkdir /etc/ssl/certs/java/ \
-  && /install-openjdk-11.sh \
+  && wget -O - https://packages.adoptium.net/artifactory/api/gpg/key/public | apt-key add - \
+  && echo "deb https://packages.adoptium.net/artifactory/deb $(awk -F= '/^VERSION_CODENAME/{print$2}' /etc/os-release) main" | tee /etc/apt/sources.list.d/adoptium.list \
+  && apt-get update \
+  && apt-get install --no-install-recommends --no-install-suggests -y temurin-11-jdk \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/* \
   ;

 # Install Google Cloud SDK
```

@macropin what do you think about using adoptium instead? I'll push this to the branch/PR if you agree.

https://adoptium.net/blog/2021/12/eclipse-temurin-linux-installers-available/